apps/condo/domains/organization/access/Organization.js

```diff
@@ -65,15 +73,16 @@ async function canReadOrganizations ({ authentication: { item: user }, listKey }
             },
             deletedAt: null,
         })
+
         const serviceOrganizationIds = uniq(billingContexts
             .map(({ organization }) => organization )
             .concat(acquiringContexts.map(({ organization }) => organization )))
             .concat(bankIntegrationOrganizationContext.map(({ organization }) => organization))
+            .concat(get(accessFilterForB2BAppServiceUser, 'id_in', []) || [])
```

Comment:
What will happen, if something different will be returned here?

Reply:
try to get `id_in` from filter, otherwise you will get an empty array (in other words, nothing will be added in result array).
Filter (B2BAppServiceUser) can return only something like `{ id_in: [1,2,3] }` or `false`